### PR TITLE
update version v1.4.1(280)

### PR DESCRIPTION
**Pull request by jiangzhiguo1992**
1. Add a new private group.
2. Added the function of sliding left and right when browsing multiple pictures/videos.
3. Added sharing and pull-down exit functions in the picture/video details page.
4. Optimize the stability and fault tolerance of file transfer.
5. Optimized the situation that the Apple end could not receive notifications.
6. Fix the problem that the client fails to link in some regions.
7. Fix the problem that the thumbnail image download fails for file transfer.
8. Fix the inconsistency between session summary and message list.
9. Fixed the problem of flashback caused by incorrect wallet address verification.
10. Increase the acceleration function of all transactions, and related settings interface.
11. Add the function of secondary confirmation of picture selection, and increase the multi-selection function of picture selection.
12. Remove unnecessary phone permissions on Android.

### CODE REVIEW

File: android/app/src/main/kotlin/org/nkn/mobile/app/push/APNSPush.kt [hunk 1]
```diff
@@ -7,17 +7,17 @@ import com.clevertap.apns.Notification
...
-
+import java.util.*
 
 /**
  * Created by JZG on 2021/6/28.
  */
 class APNSPush {
     companion object {
-        private const val ApnsTopic = "org.nkn.nmobile"
+        private const val ApnsTopic = "com.xxx.xxx"
```
Comment (iheron) [inline android/app/src/main/kotlin/org/nkn/mobile/app/push/APNSPush.kt:20]: Add in config file

File: ios/Classes/Push2/APNSPusher.swift [hunk 2]
```diff
@@ -6,10 +6,12 @@
 //
 
 import Foundation
+import Sentry
 
+let pushTopic = "com.xxx.xxx"
```
Comment (iheron) [inline ios/Classes/Push2/APNSPusher.swift:11]: Add in config file

File: android/app/src/main/kotlin/org/nkn/mobile/app/push/APNSPush.kt (hunk 1 above)
Comment (jiangzhiguo1992) [reply android/app/src/main/kotlin/org/nkn/mobile/app/push/APNSPush.kt:20]: #145

File: ios/Classes/Push2/APNSPusher.swift (hunk 2 above)
Comment (jiangzhiguo1992) [reply ios/Classes/Push2/APNSPusher.swift:11]: #145

**Review by iheron — APPROVED** [review]

**Pull request merged**